test/libsolidity/InlineAssembly.cpp

```diff
@@ -140,7 +140,7 @@ void parsePrintCompare(std::string const& _source, bool _canWarn = false)
 	if (_canWarn)
 		BOOST_REQUIRE(!Error::containsErrors(stack.errors()));
 	else
-		BOOST_REQUIRE(stack.errors().empty());
+		BOOST_REQUIRE(!stack.hasErrors());
```

Comment:
This one was intentional. Note the `_canWarn` variable. If we're in the `else` branch, we don't expect any warnings.